warewulf.spec.in

```diff
@@ -1,5 +1,11 @@
 %global wwgroup warewulf
-%{!?wwshared: %global wwshared %{_localstatedir}}
```

Comment:
This won't work they way most people want it to. When I configure changes in make, I'm looking for a resulting specfile that replicates those changes. Every line under "make genconfig" should have a @VARIABLE@ value for replacement. This allows me to send or use a custom specfile.
You also want the full URL to the source, so admins can use spectool to download it.

Reply:
I'm not sure that abides by the RPM best practices. Generally speaking, we should always be using RPM macro definitions for file placement. If you can think of a way to conditionally do that so the OHPC paths are figured out automatically, I'd love a PR for that (I'm not sure what was there would do that after all of my other changes).

Regarding the full URL to the source, that has been fixed.